orchestration/templates/cron-workflow.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: argoproj.io/v1alpha1
+kind: CronWorkflow
+metadata:
+  name: clinvar-ingest
+spec:
+  suspend: {{ not .Values.cron.enable }}
+  schedule: {{ .Values.cron.schedule | quote }}
```

Comment:
The value in values.yaml is a string; is this going to wrap the string in double quotes? Aka like double-wrapping it? Or does this just work somehow lol

Reply:
This is a slightly-longer-but-hopefully-clearer equivalent to:
```
schedule: '{{ .Values.cron.schedule }}'
```